packages/pds/src/index.ts

```diff
@@ -19,9 +19,10 @@ import { ServerConfig, ServerConfigValues } from './config'
 import { Locals } from './locals'
 import { ServerMailer } from './mailer'
 import { createTransport } from 'nodemailer'
-import SqlMessageQueue from './db/message-queue'
+import SqlMessageQueue from './stream/message-queue'
```

Comment:
just a nit, but I kinda want a different name for this than `stream` that is more descriptive of its function. `stream` just makes me think of data streams not event processing